identity_account_storage/src/types/did_type.rs

```diff
@@ -0,0 +1,8 @@
+// Copyright 2020-2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+/// Supported types of DID that can be generated.
+#[derive(Clone, Copy, Debug)]
+pub enum DIDType {
+  IotaDID,
```

Comment:
It might be necessary to add associated values to the `DIDType` variants.
E.g.

```rust
pub enum DIDType {
  IotaDID(network: NetworkName),
  KeyDID,
  ...
}
```

Not worth it now though because we might refactor `Storage` for the Stardust method.